app/controllers/catalog_controller.rb

```diff
@@ -114,8 +114,9 @@ def self.chf_search_fields
     # solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
     config.add_index_field solr_name("title", :stored_searchable), label: "Title", itemprop: 'name', if: false
+    config.add_index_field solr_name("additional_title", :stored_searchable), label: "Additional Title", itemprop: 'name', if: :present?
```

Comment:
But you're right it was an accident to leave it as a duplicate of the one from 'title', that was a copy paste error. I guess I'll change it to "additional_title", I dunno.  I could spend time figuring out what hydra intends it to be doing and if it's doing that and if not why not, but I could spend many weeks doing that for all parts of hydra, ya know?